shared/templates/template_BASH_file_contents

```diff
@@ -0,0 +1,8 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+cat << EOF > {{{ FILEPATH }}}
```

Comment:
Should this be
```
cat << 'EOF' > {{{ FILEPATH }}}
```
?

I believe this whole approach is inherently fragile because of potential escaping issues.

Reply:
Thank you for pointing the potential problem with escaping.
Indeed, having the actual file contents in the datastream can open up another can of worms.

We already have a similar situation with the login banners, where 1) the banner needs to be escaped for inclusion xml 2) and also escaped for bash remediation, but 3 ) the same escaping breaks Ansible remediation.

Reply:
The variable expansion issue has been fixed in https://github.com/ComplianceAsCode/content/pull/5110/commits/1bb775638cd694750016b8ecb567e7f3e2b103a2.

It seems we will go the fragile way for now, hopefully not for long.